toolflow/vivado/platform/ultra96v2/ultra96v2.tcl

```diff
@@ -1,6 +1,5 @@
-#!/bin/bash
 #
-# Copyright (C) 2014 Jens Korinth, TU Darmstadt
+# Copyright (C) 2020 David Eckhardt TU Darmstadt
```

Comment:
We should maybe change this to something ESA related (we should add copyright headers to all files anyway...). The original file was never by Jens as I built it but never bothered to change it.

Even if we keep it this way, DE can't simply replace the old notice with his own one as a single one.

Reply:
That is an error from git, it tries to match it with a previously existing file. Because of the large part of copyright, it thinks it was a file rename and not a new file. Ultra96 was never there before.